www/activities/foods-meals-recipes/js/foods-meals-recipes.js

```diff
@@ -421,7 +421,7 @@ app.FoodsMealsRecipes = {
     return 0;
   },
 
-  renderItem: async function(data, el, checkbox, sortable, clickable, clickCallback, tapholdCallback, checkboxCallback, timestamp, thumbnailSetting) {
+  renderItem: async function(data, el, checkbox, sortable, clickable, clickCallback, tapholdCallback, checkboxCallback, timestamp, thumbnailSetting, showBrand) {
```

Comment:
The renderItem function is called in many places throughout the app, not just from the diary. To make sure the behavior doesn't change in these other places, you need to pass `true` for the showBrand parameter there.

Also I know this is nitpicking, but I think it would be better to put the new showBrand parameter next to the existing timestamp parameter instead of at the end.

Reply:
I think the number of parameters in this function is getting out of hand :) It will need to be refactored at some point so I don't suppose it matters too much what position this new parameter has. But if you prefer it next to timestamp I won't complain :)

Reply:
@EmilJunker : Will also take this into account. 
@davidhealey : I probably won't try to refactor anything at this point, sorry ^_^"